packages/subiquity_client/test/subiquity_client_test.dart

```diff
@@ -110,7 +110,7 @@ void main() {
       expect(await client.getVariant(), equals(Variant.DESKTOP));
     });
 
-    test('source', () async {
+    test('applicationsSelection', () async {
```

Comment:
Could we keep the 'source' name here, as it's consistent with subiquity's method and directory names?